index.d.ts

```diff
@@ -77,6 +77,8 @@ export interface IDataTableProps<T> {
     selectedCount: number;
     selectedRows: T[];
   }) => void;
+  persistSelectedRowsOnSortChange?: boolean;
```

Comment:
I really hate these long property names and having to manage them, (it's my fault 😉 ). When I created this lib I didn't envision it growing like this. 

I'm planning on a next version(v7 or v8) to move some of these to an options object so this would be a good start

paginationServerOptions: { persistSelectedOnSort: bool }

same probably for `persistSelectedRowsOnPageChange` below. 

We can set the defaults in the proptypes file


paginationServer
paginationServerOptions

Reply:
I totally agree. Modified to the following:
```
paginationServerOptions: {
  persistSelectedOnSort: bool,
  persistSelectedOnPageChange: bool,
}
```

`onPageChange` is more meaningful than `onPage`, so I went with that. If you're not fond of that though I can change it.